src/components/shared/DateAndDebit/DateAndDebit.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "styles/theme";
+
+.layoutDateAndDebit {
+  display: flex;
+  justify-content: space-between;
+  width: inherit;
+  border-top: 1px solid $black-haze;
```

Comment:
I'd suggest standardising the borders, and putting named borders into your theme, rather than inlining borders in specific style sheets. This makes management easier, and helps with styling consistency.